channels/us.m3u

```diff
@@ -1071,3 +1071,13 @@ https://thegateway.app/YouToo/CueTones/playlist.m3u8
 https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/c3Z5i2L.png" group-title="",Z Living
 https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r_prod/tracks-v1a1/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US;UK" tvg-language="English" tvg-logo="https://i.imgur.com/pggeczF.jpg" group-title="Sport",World of Free Sports
+https://a.jsrdn.com/broadcast/ab14783a09/+0000/high/c.m3u8
```

Comment:
Again, cannot really tell if this channel is really from the US...

Found one YouTube channel with the same name that claims to be German.

I also found a UK channel called FreeSports that sometimes broadcasts a program called "World of Free Sports"...

Could be someone that compilated all of those and had put it into a stream.

Reply:
this is a distro.tv channel. Distro.tv is based in US and targets its channels to UK too.

Reply:
Indeed.